# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/trusted-types/TrustedTypePolicyFactory-createPolicy-nameTests.html.ini
# Commit: 4e3e9ade5089
# Full Hash: 4e3e9ade50894ab253b5e02423e59a4d0df1503c
# Author: Mirko Brodesser <mbrodesser@igalia.com>
# Date: 2024-06-19 21:39:42
# Regressor Bug: 1901491
# File Overlap Count: 1
# Description:
#   Bug 1901491: part 3) Implement `ShouldTrustedTypePolicyCreationBeBlockedByCSP` for Windows (not Workers) including reporting exceptions, excluding reporting violations. r=tschuster,webidl,saschanaz
#   
#   Parsing the trusted-types directive's values is not added to
#   `nsCSPParser::sourceList` because the latter supports keywords not
#   supported by the trusted-types directive. E.g. 'report-sample'.
# ==============================================================================

diff -r dd03de7652a1 -r 4e3e9ade5089 testing/web-platform/meta/trusted-types/TrustedTypePolicyFactory-createPolicy-nameTests.html.ini
--- a/testing/web-platform/meta/trusted-types/TrustedTypePolicyFactory-createPolicy-nameTests.html.ini	Wed Jun 19 12:51:28 2024 +0000
+++ b/testing/web-platform/meta/trusted-types/TrustedTypePolicyFactory-createPolicy-nameTests.html.ini	Wed Jun 19 12:51:29 2024 +0000
@@ -1,4 +1,5 @@
 [TrustedTypePolicyFactory-createPolicy-nameTests.html]
   prefs: [dom.security.trusted_types.enabled:true]
-  [duplicate policy name attempt throws]
+  [Error messages for duplicates and unlisted policies should be different]
+    # See <https://github.com/w3c/trusted-types/issues/511>.
     expected: FAIL